prow/cmd/tackle/main.go

```diff
@@ -197,22 +309,23 @@ func currentClusters(proj string) (map[string]cluster, error) {
 func createCluster(proj, choice string) (*cluster, error) {
 	const def = "prow"
 	if choice == "" {
-		fmt.Printf("Cluster name [%s]: ", def)
-		fmt.Scanln(&choice)
-		if choice == "" {
-			choice = def
-		}
+		choice = prompt("Cluster name", def)
 	}
 
-	cmd := exec.Command("gcloud", "container", "clusters", "create", choice)
+	zone, err := selectZone()
+	if err != nil {
+		return nil, fmt.Errorf("get current zone: %v", err)
```

Comment:
nit: The error might not be related to getting the current zone.

Reply:
@cjwagner - what is a better error message here?
e.g.
```golang
return nil, fmt.Errorf("select current zone: %v", err)
```

Reply:
/shrug
It doesn't matter much I was just thinking the use of "current" is a bit misleading since we're selecting a zone for a new cluster.